Fix building with ICU 68.

ICU 68 no longer defines the TRUE macro.

https://github.com/GNOME/libxml2/pull/24

diff --git a/encoding.c b/encoding.c
index c34aca44..264f60bb 100644
--- a/encoding.c
+++ b/encoding.c
@@ -2004,7 +2004,7 @@ xmlEncOutputChunk(xmlCharEncodingHandler *handler, unsigned char *out,
 #ifdef LIBXML_ICU_ENABLED
     else if (handler->uconv_out != NULL) {
         ret = xmlUconvWrapper(handler->uconv_out, 0, out, outlen, in, inlen,
-                              TRUE);
+                              1);
     }
 #endif /* LIBXML_ICU_ENABLED */
     else {
